WBooks/AppDelegate.swift

```diff
@@ -23,8 +23,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.rootViewController = UINavigationController(rootViewController: WBooksTabBarController())
         window?.makeKeyAndVisible()
         
+        setStatusBarStyle()
+        
         return true
     }
+    
+    func setStatusBarStyle() {
+        UINavigationBar.appearance().setBackgroundImage(UIImage(),for: .default)
```

Comment:
I think this will change the navigation bar appearance in the whole app but you only need to change the style of the main navigation bar. Lets find another way to do this.